backend/cfg/cfg.ml

```diff
@@ -377,7 +377,7 @@ let can_raise_operation : operation -> bool = function
   | Floatofint -> false
   | Intoffloat -> false
   | Probe _ -> true
-  | Probe_is_enabled _ -> true
+  | Probe_is_enabled _ -> false
```

Comment:
While `Probe` can raise and is not pure, `Probe_is_enabled` just reads a global and therefore cannot raise and is pure (it is fixed as part of #785), similarly to Load. Instead of the change you propose here, we should fix `Cfg.can_raise_operation Probe_is_enabled`  to `false`, which would also match the result of `Mach.operation_can_raise` in backend and asmcomp subdirectory. Thanks for catching that they are different!

Reply:
But the change I propose here is exactly `Cfg.can_raise_operation Probe_is_enabled` to `false`.
It is in #785 because I merged #783 into it to avoid unnecessary refactor of liveness.
And #783 has it because otherwise it's broken.

Reply:
But I put it as another commit because it's much more urgent than the other matters.

Reply:
oh, right - clearly I was confused!